plugin/goto_diagnostic.py

```diff
@@ -221,10 +228,11 @@ def _project_path(self, parsed_uri: ParsedUri) -> str:
         return path
 
 
-class DiagnosticInputHandler(sublime_plugin.ListInputHandler):
+class DiagnosticInputHandler(PreselectedListInputHandler):
```

Comment:
This should not be a PreselectedListInputHandler. I mean it wouldn't break anything, but it also makes no sense if it's always gonna be instantiated without a initial value to preselect, so it would just be confusing here. PreselectedListInputHandler only makes sense if it's not the last InputHandler on the stack.